src/subscription_manager/ga_loader.py

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (c) 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# Red Hat trademarks are not licensed under GPLv2. No permission is
+# granted to use or replicate Red Hat trademarks that are incorporated
+# in this software or its documentation.
+#
+
+import imp
+import os
+import sys
+
+import logging
+log = logging.getLogger('rhsm-app.' + __name__)
+
+from subscription_manager import version
+
+
+class GaImporter(object):
```

Comment:
Could we get some good comments added to this guy, this looks like the real guts of the feature.